### PR TITLE
chore: Django bump to <=5.0

### PR DESCRIPTION
**Pull request by xdaniel3**

### CODE REVIEW

File: pynotify/models.py [hunk 1]
```diff
@@ -238,6 +238,9 @@ def clean_extra_data(self):
         return self._clean_json_value(self.extra_data)
 
     def clean(self):
+        if self.is_adding:
+            return set()
```
Comment (Formulka) [inline pynotify/models.py]: it's a bit weird that it usually doesn't return anything

**Review by Formulka — APPROVED** [review]

File: pynotify/models.py (hunk 1 above)
Comment (rubickcz) [reply pynotify/models.py]: I think it is better close the code below into if statement, rather then introducing guard clause here. I.e:
```
if not self.is_adding:
   keys = ...
```

**Review by HonzaSaibic — APPROVED** [review]

File: pynotify/models.py (hunk 1 above)
Comment (xdaniel3) [reply pynotify/models.py]: yeah, returning set() is wrong, good catch 👍 However I do not like negative conditions if they are not necessary, they are harder to read generally. So I changed to
```
if self.is_adding:
    return
```

File: setup.py [hunk 2]
```diff
@@ -14,8 +14,8 @@
 requirements = [
     'beautifulsoup4 >=4.8.0',
     'celery >= 4.2.0',
-    'django ~= 3.0',
-    'skip-django-chamber >= 0.6.16.3',
+    'django>=4.2',
+    'skip-django-chamber @ git+https://github.com/skip-pay/django-chamber@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
    'skip-django-chamber>=0.7.2',
```

**Pull request merged**